cosmos/models/Cosmos.py

```diff
@@ -31,9 +31,10 @@ def default_get_submit_args(task, parallel_env='orte'):
         time = ' -W 0:{0}'.format(task.time_req) if task.time_req else ''
         return '-R "{rusage}span[hosts=1]" -n {task.core_req}{time}{queue} -J "{jobname}"'.format(**locals())
     elif task.drm in ['ge', 'drmaa:ge']:
-        return '-cwd -pe {parallel_env} {core_req}{priority} -N "{jobname}"{queue}'.format(
+        return '-cwd -pe {parallel_env} {core_req}{priority} -N "{jobname}"{job_class}{queue}'.format(
             priority=' -p %s' % default_job_priority if default_job_priority else '',
-            queue=' -q %s' % task.queue or '',
```

Comment:
I think this was meant to not add a `-q ...` to the output if task.queue is unset, but that's not what it was doing